tests/acceptance/standard/test_hooks_processor.py

```diff
@@ -0,0 +1,115 @@
+import logging
+import textwrap
+import time
+
+import gitlab
+from gitlab.v4.objects import ProjectHook
+
+from gitlabform import GitLabForm
+from gitlabform.gitlab import GitlabWrapper
+from gitlabform.gitlab.core import NotFoundException
+
+from tests.acceptance import (
+    allowed_codes,
+    create_group,
+    delete_groups,
+    create_project,
+)
+
+logger = logging.getLogger(__name__)
+
+
+test_yaml = """
+      config_version: 3
+      projects_and_groups:
+        "*":
+          project_settings:
+            visibility: internal
+        test_group/mystery:
+          project_settings:
+            rick: pickle
+          hooks:
+            "http://plumbus.org/create":
+                push_events: true
+                merge_requests_events: true
+            "http://birdman.chirp/update":
+                push_events: true
+      """
+
+delete_yaml = """
+      config_version: 3
+      projects_and_groups:
+        "*":
+          project_settings:
+            visibility: internal
+        test_group/mystery:
+          project_settings:
+            rick: pickle
+          hooks:
+            "http://plumbus.org/create":
+                delete: true
+      """
+
+first_hook_dict = {"url": "http://birdman.chirp/update", "push_events": True}
+
+second_hook_dict = {
+    "url": "http://plumbus.org/create",
+    "push_events": True,
+    "merge_requests_events": True,
+}
+
+
+def test_hooks_processor():
+    config = textwrap.dedent(test_yaml)
+    gf = GitLabForm(
+        include_archived_projects=False,
+        target="test_group/mystery",
+        config_string=config,
+    )
+    gl = GitlabWrapper(gf.gitlab).get_gitlab()
```

Comment:
You don't need to import and use `GitlabWrapper` class. All the existing acceptance tests were already migrated to use the python-gitlab library. Just the app logic wasn't migrated yet. Please take a look at the existing tests.

Reply:
sorry, I moved my test a couple of times, didn't clean up properly.